src/api/spotify.ts

```diff
@@ -0,0 +1,12 @@
+export function getUserData() {
+	const xhr = new XMLHttpRequest();
+	xhr.open('GET', `https://api.spotify.com/v1/me`, true);
+	xhr.setRequestHeader('Content-Type', 'application/json');
+	xhr.setRequestHeader('Authorization', `Bearer ${localStorage.getItem('access_token')}`);
+	xhr.send(null);
+	xhr.onload = () => {
+		const responseData = JSON.parse(xhr.responseText);
+		localStorage.setItem('display_name', responseData.display_name);
+		localStorage.setItem('user_id', responseData.id);
```

Comment:
Should we re-organize this into something neater? Save time and reply with proposals first before making code changes and another PR.

Was wondering if each value should be stored in a map instead (then stringify and setItem). If agree, then need to pick a better name for key. Should we also prepend all of our keys with `[ViBE]:` or something alike?

Reply:
What do you think about storing data as a JSON object: localStorage.setItem('ViBE', data) where
data= {
  display_name = responseData.display_name,
  user_id = responseData.id
  ...
}

Reply:
Works for me